shakedown/cli/main.py

```diff
@@ -313,7 +314,7 @@ def pytest_sessionfinish(self, session, exitstatus):
                 for output in shakedown.stdout:
                     echo(output)
 
-    opts = ['-q', '--tb=no']
+    opts = ['-q', '--tb=no', '--timeout='+str(args['timeout'])]
```

Comment:
Yes, it is used by the `pytest-timeout` plugin -- and I'll update to change to using `format`, one sec...